src/app/parcels/ActionBar/ActionModals/DayOverviewModal.tsx

```diff
@@ -13,6 +13,9 @@ const getPdfErrorMessage = (error: DayOverviewPdfError): string => {
         case "parcelFetchFailed":
             errorMessage = "Failed to fetch parcel data.";
             break;
+        case "failedToRetrieveCongestionChargeDetails":
+            errorMessage = "Failed to retrieve congestion charge data";
```

Comment:
Yeah I had this chat with Ricky! What you are suggesting was what I was originally aiming to do, but it turned out to be more difficult than expected, so that is one reason it has ended up this way - the other is Ricky said the advantage of showing an error message and no data is that the problem is more likely to be reported and fixed.  But I would be interested in finding out how to make this happen

Reply:
as in, I think generally the idea is just showing an error message on any error is the standard, but I also think the reason I have done that here is because I couldn't work out how to do it

Reply:
Do you know why is it difficult? Take this from someone who doesn't know much about this codebase lol but I'd probably start in `DayOverviewPdfButton.tsx` where you return `congestionChargeError`, and instead of returning there try to pass that error as a new property onto the PDF (e.g. `listOfNonFatalErrors`). Then in the `DayOverviewPdf.tsx` go over the errors, and if there are any, create messages for each code and add them to the top of the file. Then when you do 
```
 {parcel.collection_centre!.name === "Delivery" &&
                    parcel.congestionChargeApplies && (
                        <FontAwesomeIconPdfComponent faIcon={faCopyright} color="red" />
                    )}
```  
I don't know the exact syntax now and you might need to put in some effort to make it nice (perhaps make a new component?) but aim to say something like 
`if collection_cnetre.name === delivery
   if congestionChargeError in listOfNonFatalErrors
      use grey ? icon
   else if congestionChargeApplies
      use red C icon
`
```

Don't spend too much time on it if it doesn't work and leave it as is, but if you'd like to try hopefully some of these ideas can help

Not sure if I agree with "problem will get reported faster and fixed if you return nothing" - it will probably get reported very fast yes, but not necessarily fixed fast 😄 and if it's not something that gets fixed by a refresh and needs actual code changes I don't think people using the system would be happy to not be able to print their list for a while so I think leaning towards a more gentle fail might be better, but I guess it depends who you ask